201-vm-linux-jupyterhub/azuredeploy.json

```diff
@@ -0,0 +1,364 @@
+            }
+        },
+        "_artifactsLocation": {
+            "type": "string",
+            "metadata": {
+                "description": "The base URI where artifacts required by this template are located. When the template is deployed using the accompanying scripts, a private location in the subscription will be used and this value will be automatically generated."
+            },
+            "defaultValue": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/201-vm-linux-jupyterhub/"
+        },
+        "_artifactsLocationSasToken": {
+            "type": "securestring",
+            "metadata": {
+                "description": "The sasToken required to access _artifactsLocation.  When the template is deployed using the accompanying scripts, a sasToken will be automatically generated."
+            },
+            "defaultValue": ""
+        },
+        "diskSizeGB":{
+            "type": "string",
+            "metadata":{
+                "description": "The size for an atached disk."
+            },
+            "defaultValue": "1024",
+            "allowedValues":[
+                "1024",
+                "2048",
+                "4096",
+                "8192",
+                "16384",
+                "32767"
+            ]
+        }
+    },
+    "variables": {
+        "networkInterfaceName": "[concat(parameters('vmName'),'NetInt')]",
+        "publicIpAddressName": "[concat(parameters('vmName'), 'PublicIP' )]",
+        "subnetRef": "[resourceId('Microsoft.Network/virtualNetworks/subnets', parameters('virtualNetworkName'), parameters('subnetName'))]",
+        "nsgId": "[resourceId('Microsoft.Network/networkSecurityGroups', parameters('networkSecurityGroupName'))]",
+        "storageAccountName": "[concat('storage', uniqueString(resourceGroup().id))]",
+        "storageAccountType": "Standard_LRS",
+        "storageAccountKind": "Storage",
+        "vmSize": {
+            "CPU-8GB": "Standard_F4s_v2",
+            "CPU-14GB": "Standard_DS3_v2",
+            "CPU-16GB": "Standard_D4s_v3",
+            "CPU-28GB": "Standard_DS4_v2",
+            "CPU-32GB": "Standard_F4s_v2",
+            "CPU-64GB": "Standard_D16s_v3",
+            "CPU-112GB": "Standard_DS14-4_v2",
+            "CPU-128GB": "Standard_E16s_v3",
+            "CPU-256Gb": "Standard_E32_v3",
+            "CPU-432Gb":"Standard E64_v3",
+            "GPU-56GB": "Standard_NC6_Promo",
+            "GPU-112GB":"Standard_NV12",
+            "GPU-224GB":"Standard_NV24"
+
+        },
+        "vmName": "[concat(parameters('vmName'),'-', parameters('cpu-gpu'))]",
+        "dataDisks": {
+            "lun": 0,
+            "createOption": "attach",
+            "caching": "None",
+            "writeAcceleratorEnabled": false,
+            "id": null,
+            "name": "[concat('Datadisk', parameters('vmName'))]",
+            "storageAccountType": null,
+            "diskSizeGB": null    
+        },
+        "dataDiskResources": {
```

Comment:
this is never used

Reply:
it should be my mistake.. just PR a new version which creates the date disk and attached it as a lun